lib/Driver/UnixToolChains.cpp

```diff
@@ -158,10 +158,15 @@ bool toolchains::GenericUnix::addRuntimeRPath(const llvm::Triple &T,
 
   // Honour the user's request to add a rpath to the binary.  This defaults to
   // `true` on non-android and `false` on android since the library must be
-  // copied into the bundle.
+  // copied into the bundle. An exception is made for the Termux app as it
+  // builds and runs natively like a Unix environment on Android.
+#if defined(__TERMUX__)
+  bool android_rpath = true;
+#else
+  bool android_rpath = !T.isAndroid();
+#endif
```

Comment:
This change is modifying more than it seems to: any Swift compiler build for Termux will use `true`, even when cross-compiling to pure Android.

Reply:
Yep, that is what I mean for it to do. Since this is only the default, one can always supply `-no-toolchain-stdlib-rpath` and override this default on Termux if wanted.